lib/tasks/cf.rake

```diff
@@ -0,0 +1,8 @@
+namespace :cf do
+  desc "Only run on the first application instance"
+  task on_first_instance: :environment do
+    # We expect this information to be always available or break otherwise
+    instance_index = JSON.parse(ENV["VCAP_APPLICATION"])["instance_index"]
```

Comment:
Probably didn't mean to remove this `instance_index = -1`?

Reply:
I actually did because otherwise I get:
`lib/tasks/cf.rake:4:5: W: Lint/UselessAssignment: Useless assignment to variable - instance_index. (https://rubystyle.guide#underscore-unused-vars)
    instance_index = -1
    ^^^^^^^^^^^^^^
`